### PR TITLE
HTML, CSS and Icons (images) for Portfolio

### PR DESCRIPTION
**Pull request by trace-kadenyi**
- Add index.html file with the body of the portfolio (header and headline sections).
- Add CSS stylesheet to style the portfolio.
- Add images to represent the icons of the portfolio such as menu bar and social media contacts.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+</head>
+<body>
+    <section class="mobile-skeleton">
+        <header class="header">
+            <nav class="navbar">
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
+                    <li><img src="images/menu.png" alt="menu"></li>
+                </ul>
+            </nav>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please only the hamburger icon should be inside the `nav tag.` not the logo

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+        </section>
+        <section class="contacts">
+                <img class="first" src="images/tweet.svg" alt="twitter">
+                <img src="images/the-i.svg" class="linked1" alt="letter_i">
+                <img src="images/the-n.svg" class="linked2" alt="letter_n">
+                <img class="first" src="images/the-m.svg" alt="letter_m">
+                <img class="first" src="images/cartoon.svg" alt="cartoon">
+                <img class="first" src="images/peace.svg" alt="peace">
+        </section>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please try to make the whole headline part have one` section tag` container.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+                <p>LET'S CONNECT</p>
+            </div>
+        </section>
+        <section class="contacts">
+                <img class="first" src="images/tweet.svg" alt="twitter">
+                <img src="images/the-i.svg" class="linked1" alt="letter_i">
+                <img src="images/the-n.svg" class="linked2" alt="letter_n">
+                <img class="first" src="images/the-m.svg" alt="letter_m">
+                <img class="first" src="images/cartoon.svg" alt="cartoon">
+                <img class="first" src="images/peace.svg" alt="peace">
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please ensure that your social media icons are inside a `ul tag` and each of them inside a `li tag`.
- [x] Add link place holders to each icon ..`<a href="/"></a>` .you can check how to add that [here](https://www.w3schools.com/html/html_links.asp)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
+                    <li><img src="images/menu.png" alt="menu"></li>
+                </ul>
+            </nav>
+        </header>
+        <section class="headline">
+            <img src="images/headline.png" alt="headline">
+            <div class="main">
+                <p class="intro">I'm Kadenyi Glad to see you!</p>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] kindly put this main title inside an `h1 tag`

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet">
+    <title>Portfolio: Mobile Version Skeleton</title>
+</head>
+<body>
+    <section class="mobile-skeleton">
+        <header class="header">
+            <nav class="navbar">
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
```
Comment (tufoinnkuo10) [inline index.html]: - [ optional ] you can put your name in place of `my logo` since you don't have a logo yet.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+                </ul>
+            </nav>
+        </header>
+        <section class="headline">
+            <img src="images/headline.png" alt="headline">
+            <div class="main">
+                <p class="intro">I'm Kadenyi Glad to see you!</p>
+                <p class="pitch">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
```
Comment (tufoinnkuo10) [inline index.html]: - [x] please try to follow the figma characteristics of exact margins, font weight, line height, sizes, family, etc so it looks similar.
- your design 👇🏾 
Screenshot (48)_LI
- figma design 👇🏾 
Screenshot (39)_LI

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @TracK92 

Great job so far with your progress 😄 . There are some issues that you still need to work on to go to the next project but you are almost there!

**To Highlight**
✅ You used the correct Github flow
✅ Good commit messages
✅ linter checks validated
✅ Good pull request summary
✅ Correct Readme template

Required Changes ♻️
Check the comments under the review

**Optional suggestions**
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by trace-kadenyi** [comment]
Dear Reviewer, 

Thank you very much for your detailed review and instructions on how to amend my code. I have made all the recommended changes. I was a bit confused about the suggestion to remove the Logo from the navbar but I followed your instructions. Kindly examine my reviewed code.

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️
**Hi @TracK92,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉


You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by trace-kadenyi** [comment]
Fweuks! Thank you [Kossi-stack](https://github.com/Kossi-stack).

**Pull request merged**